src/applications/medical-copays/components/HTMLStatementList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation, Link } from 'react-router-dom';
+import moment from 'moment';
+import { sortStatementsByDate } from '../utils/helpers';
+
+const HTMLStatementList = () => {
+  const { pathname } = useLocation();
+  const selectedId = pathname.replace('/balance-details/', '');
+  const statements = useSelector(({ mcp }) => mcp.statements) ?? [];
+  // get selected statement
+  const [selectedCopay] = statements?.filter(({ id }) => id === selectedId);
```

Comment:
Do we know when statements would be `falsey`?  I understand why the Elvis (`?.`) is here, but ifs null here it will crash down the page. 

If statements can be nothing, we should plan accordingly

Reply:
Nuked the Elivs checks, you wouldn't be able to make it to this page if statements were falsey